js/windowMgr.js

```diff
@@ -1304,6 +1316,43 @@ const logBrowserWindowEvents = (browserWindow, windowName) => {
 
 };
 
+/**
+ * Validates the network by fetching the pod url
+ * every 5sec, on active reloads the given window
+ *
+ * @param window
+ * @param url
+ */
+const isSymphonyReachable = (window, url) => {
+    if (!networkStatusCheckIntervalId) {
```

Comment:
let's simplify the if blocks, let's invert the conditions to return for a cleaner codebase.